src/Sylius/Bundle/ThemeBundle/Configuration/Filesystem/FilesystemConfigurationSourceFactory.php

```diff
@@ -33,7 +33,7 @@ public function buildConfiguration(ArrayNodeDefinition $node): void
                     ->scalarNode('filename')->defaultValue('composer.json')->cannotBeEmpty()->end()
                     ->integerNode('scan_depth')->info('Restrict depth to scan for configuration file inside theme folder')->defaultNull()->end()
                     ->arrayNode('directories')
-                        ->defaultValue(['%kernel.project_dir%/app/themes'])
+                        ->defaultValue(['%kernel.project_dir%/themes'])
```

Comment:
We can't change the default value, it's a BC break for anyone already using themes and updating to the newest release. Could you revert that change? We can add a snippet to the docs to configure it like this (I've posted it above).